lib/evss/gi_bill_status/service.rb

```diff
@@ -8,7 +8,11 @@ class Service < EVSS::BaseService
 
       def get_gi_bill_status
         raw_response = get ''
-        EVSS::GiBillStatus::GiBillStatusResponse.new(raw_response)
+        EVSS::GiBillStatus::GiBillStatusResponse.new(raw_response.status, raw_response)
+      rescue Faraday::ParsingError
+        EVSS::GiBillStatus::GiBillStatusResponse.new(403)
```

Comment:
Now that we're rescuing, we need to log to sentry.  Make use of the `SentryLogging` mixin.  

As a special request, could you also add the `BASE_URL` as extra context?  In the past when a 404 has occurred, it's been difficult to know _exactly_ where we were reaching out to since `BASE_URL` is a combination of some devops shenanigans and `vets-api` settings.